constants.py

```diff
@@ -0,0 +1,118 @@
+class commands:
```

Comment:
As I've write into the comments, I'm not agree with this part.
May be we can think about some configuration.ini check.

Reply:
Understood. I will rollback to previous commit and implement checking function into it.